app/utils/post/index.ts

```diff
@@ -71,3 +71,15 @@ export const sortPostsByNewest = (posts: PostModel[]) => {
         return -1;
     });
 };
+
+export const processPostsFetched = (data: PostResponse) => {
```

Comment:
this is fine, I wonder why we kept the previous approach for threads but made this change for posts.

Reply:
I changed it based on the feedback as it doesn't server the same purpose as earlier, I removed the `storing in db` part and just using it for mapping the response.